qa/rpc-tests/p2p-acceptblock.py

```diff
@@ -279,8 +279,14 @@ def run_test(self):
 
         # 7. Send the missing block for the third time (now it is requested)
         test_node.send_message(msg_block(blocks_h2f[0]))
-
         test_node.sync_with_ping()
+
+        while self.nodes[0].getblockcount() != 290:
+            time.sleep(1)
+            j = j + 1
+            if (j > 60):
```

Comment:
I added a comment there that we have to wait for the reorg to compete but the 60 seconds is just a number i picked...it has to break at some point to throw an assert if the reorg isn't happening.  The reorg should only take a second or two to complete but on my system, it's a bit slow and always fails if i don't wait a second or two...but rather than put in a hard limit i thought the loop would be better.